deploy/crds/apps.openshift.io_servicebindingrequests_crd.yaml

```diff
@@ -252,11 +253,28 @@ spec:
         status:
           description: ServiceBindingRequestStatus defines the observed state of ServiceBindingRequest
           properties:
-            applicationObjects:
+            applications:
               description: ApplicationObjects contains all the application objects
                 filtered by label
               items:
-                type: string
+                description: BoundApplication defines the application workloads to
+                  which the binding secret has injected.
+                properties:
+                  group:
+                    type: string
+                  kind:
```

Comment:
For applications, we call it a `resource` in `gvr` format. Shall we change it to resource to maintain consistency?

Reply:
Yeah, I did consider that. However, the https://godoc.org/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured API didn't help me with the `Resource`.

Reply:
https://github.com/redhat-developer/service-binding-operator/blob/3d6079770b849631744f67b9dddb1a1baf3cf8af/pkg/apis/apps/v1alpha1/servicebindingrequest_types.go#L72

Reply:
The code where we were fetching the retrieved app's details didn't have the `resource` information.

https://github.com/redhat-developer/service-binding-operator/pull/363/files#diff-2e81f3f412c3c3df6c39c5bcbce41026R278 `obj` doesn't provide the `Resource` metadata.